xarray/coding/cftimeindex.py

```diff
@@ -476,6 +476,24 @@ def to_datetimeindex(self, unsafe=False):
                 'dates.'.format(calendar), RuntimeWarning, stacklevel=2)
         return pd.DatetimeIndex(nptimes)
 
+    def strftime(self, date_format):
+        """
+        Return a pd.Index of formatted strings specified by date_format, which
+        supports the same string format as the python standard library. Details
+        of the string format can be found in `python string format doc
+        <https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior>`__
+
+        Parameters
+        ----------
+        date_format : str
+            date format string (e.g. "%Y-%m-%d")
+
+        Returns
+        -------
+        ndarray of formatted strings
```

Comment:
Wasn't sure whether it's better to return a `np.array` or a `pd.Index`.  Thoughts either way? If this method is even desired here.